Framework/DataHandling/src/LoadPSIMuonBin.cpp

```diff
@@ -179,9 +179,11 @@ void LoadPSIMuonBin::exec() {
 
   double timeZero = 0.0;
   if (m_header.realT0[0] != 0) {
-    timeZero = m_header.realT0[0];
+    timeZero = static_cast<double>(*std::max_element(
+        std::begin(m_header.realT0), std::end(m_header.realT0)));
```

Comment:
Since realT0 is an array of doubles we don't need to cast here.
```suggestion
        timeZero = m_header.realT0[std::max_element(
        std::begin(m_header.realT0), std::end(m_header.realT0)]
```
Something to this effect is likely better as it avoids a cast.

Reply:
Since max_element returns a float wouldn't it still need to be cast to an int in this case?